exercises/robot/manipulation_station_io.ipynb

```diff
@@ -0,0 +1,535 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "OvSvFZH4_9Ni"
+   },
+   "outputs": [],
+   "source": [
+    "from IPython.display import HTML\n",
+    "from pydrake.systems.framework import GenerateHtml\n",
+    "\n",
+    "diagram.set_name(\"diagram\")\n",
+    "HTML('<script src=\"https://unpkg.com/gojs/release/go.js\"></script>' + GenerateHtml(diagram))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "FKDGlrCFqb15"
+   },
+   "source": [
+    "Now if we set the joint position of the Kuka arm, we should expect to get the same values from the iiwa_position_measured port, which can be found from the output ports of **manipulation_station** object. Below we demonstrate how this can be done using **drake**'s syntax. You may also find it useful to review the **system** class documentation [here](https://drake.mit.edu/pydrake/pydrake.systems.framework.html?highlight=output_port#pydrake.systems.framework.System_)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "PwWvrx1_cPLl"
+   },
+   "outputs": [],
+   "source": [
+    "# provide initial states\n",
+    "q0 = np.array([-1.57, 0.1, 0, -1.2, 0, 1.6, 0])\n",
+    "# set the joint positions of the kuka arm\n",
+    "station.SetIiwaPosition(station_context, q0)\n",
+    "# examine the output port\n",
+    "station.GetOutputPort(\"iiwa_position_measured\").Eval(station_context)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "8d7sH3kzrnYL"
+   },
+   "source": [
+    "Note that the [output port](https://drake.mit.edu/pydrake/pydrake.systems.framework.html?highlight=outputport#pydrake.systems.framework.OutputPort) named \"iiwa_position_measured\" is first retrieved from the station and then evaluated using **Eval** method. This is a very common approach to read the values of a selected output port.\n",
+    "\n",
+    "Alternatively, you may retrieve the joint angles from the **plant**, which is a subsystem of the manipulation station."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "gbpZuVa_axTY"
+   },
+   "outputs": [],
+   "source": [
+    "joint_angles = []\n",
+    "for i in range(1, 8):\n",
+    "  joint_angles.append(\n",
+    "  plant.GetJointByName('iiwa_joint_{}'.format(i)).get_angle(plant_context)\n",
+    "  )\n",
+    "\n",
+    "# alternatively, use GetPositions to obtain the generalized positions\n",
+    "# from the plant context\n",
+    "q_general = plant.GetPositions(plant_context)\n",
```

Comment:
not updated with my comment from google?

Reply:
Done. Changed to `plant.GetPosition(plant_context, plant.GetModelInstanceByName("iiwa")` and printed both angles to show folks it can be done both ways.